packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -244,6 +245,9 @@ export class GLTFExporter {
 
     private readonly _options: Required<IExportOptions>;
 
+    /**

```

Comment:
Still a few of those :-)

Reply:
Ok ok :-) However `npm run lint` didn't report anything... I think maybe we should check the prettier and eslint settings. I need to save these files in VS Code w/o formatting then run the linter manually. Do you have a different easier approach to avoid these types of formatting errors?